lib/vm.js

```diff
@@ -21,6 +21,7 @@
 
 'use strict';
 
+const { kParsingContext } = require('internal/vm');
 const binding = process.binding('contextify');
```

Comment:
This seems like over-complicating it.  You can create the symbol in InitContextify() or ContextifyScript::Init() and set it on `target` and then use it directly from the binding object, no need for a new file in lib/internal.